article/_posts/2023-02-09-fully-automated-QC-for-cardiac-CMR.md

```diff
@@ -0,0 +1,101 @@
+<!-- - https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8363616 -->
+
+## Step 2: Image segmentation
+
+Segmentation of LV, RV and LV myocardium in all frames.
+Trained on manually annotated data. Dataset contained 3975 subjects containing subjects with cardiac diseases and healthy subjects.
+
+## Step 3: Parameter calculation
+
+Steps used in parameter calculations:
+
+- Align short and long axis imaging stacks
+- Calculate LV and RV volume curves and LV mass (LVM)
+- Calculate metrics from volume curves (EDV, ESV, peak ejection rate, etc.)
+- Feature tracking with end-diastolic LV wall as ROI. Using Medical Image Registration ToolKit.
+- More metrics from feature tracking. Included Global circumferential strain, radial strain and longitudinal strain.
+
+## Step 4: Post-analysis QC (QC2)
+
+QC2 contains 3 parts in order to flag cases for clinician review.
+
+- Validation of orientation of images and missing slices
+- Inspection of output parameters; flag suspicious exams.
+    - Difference >10% on LV and RV SV
+    - Difference of volume between phases
+- SVM classification for abdnomalities in volume and strain curves.
+    - Data used for training these classifiers was from healthy subjects and subjects with cardiomyopathy from UK Biobank, annotated by an expert.
+
+# Results and Discussion
+
+Importance of QC1 lies in the significance of the impact of motion artifacts and off-axis planning on the computed metrics. These issues will not be recognisable easily in post-analysis QC, therefore are best flagged before.
+
+QC2 relies on anatomical and clinical knowledge to identify abnormalities. According to the authors, this can extend to detection of cases not anticipated in the development of the algorithms.
+
+Authors believe that including an uncertainty score in the segmentation networks could be beneficial to the CMR analysis.
+
+Pipeline validation done with comparisons with expert review showed that the QC steps correctly detected erroneous cases.
+
+
+**Rate of flagged CMRs**
+|  | Pipeline | Clinicial review  |
+|---|---|---|
+| Healthy patients| 26% | 15.2% |
+| Cardiac patients | 32% | 11% |
+
+
+It seems however that the pipeline may systematically remove unhealthy subjects based on medical condition rather than image/metric quality.
+
+Sesitivity to errors in volume curves and strain was high. However, the specificity shows that it is very sensitive to false positive cases (flags valid sequences as erroneous)
+
+![](/article/images/cardiacQCpipeline/validationResultsTable.jpg)
+
+
+# Conclusions
+
+Quality control measures allow fully autonomous processing of CMR exams, while flagging possible problematic cases for clinician review. 
+
+The pipeline can facilitates creation of a large set of reference values for characterization of cardiac function. The pipeline allows for much shorter computation time compared to semi-automated segmentation and FT commercial software. 
```

Comment:
can facilitates creation (...) for characterization -> can facilitate the creation (...) for the characterization

What is FT ?

Reply:
Done in 51d9c8d.